vtex/faststore
# Add facebook Pixel as plugin

## What's the purpose of this pull request?
Adds the facebook pixel integration as a plugin. It is very similar to https://github.com/vtex-apps/facebook-pixel. I'm still not sending the currency to FB servers (the GTM plugin also does not as well).

## How it works? 
<em>Tell us the role of the new feature, or component, in its context.</em>

## How to test it?
You can use this chrome extension to verify that the events are being sent https://chrome.google.com/webstore/detail/facebook-pixel-helper/fdgfkebogiimcoedlicjlajpkdmockpc

## References
<em>Spread the knowledge: is there any content you used to create this PR that is worth sharing?</em>  

<em>Extra tip: adding references to related issues or mentioning people important to this PR may be good for the documentation and reviewing process</em>
